### DISCUSSION TITLE
第四节课笔记(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#379] [笔记]
笔记放在这里！
作业**不放在**这里！

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135608847

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17968851

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson4/lesson4%E7%AC%94%E8%AE%B0.md

**Comment by IdealDestructor** [comment]
https://blog.deqiang.wang/2024/01/19/Project-LLM4/

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson4.md

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/note/04-Xtuner%20note.md

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135738066

**Comment by wade361** [comment]
https://github.com/wade361/-Wade/issues/5

**Comment by Cyberworldvoyager** [comment]
https://github.com/Cyberworldvoyager/Interlm_lecture/blob/main/lecture%204.pdf

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/tree/main/XTuner%20%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%8D%95%E5%8D%A1%E4%BD%8E%E6%88%90%E6%9C%AC%E5%BE%AE%E8%B0%83%E5%AE%9E%E6%88%98

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/tree/main/Lecture4
